### ISSUE TITLE
Double-check "v" prefix does not break semver comparison

### ISSUE THREAD

**Issue by davegaeddert** [dependencies-io/php#20]
See this related issue: https://github.com/dependencies-io/support/issues/27

My assumption is that this is somehow related to platform requirements (which has been the case before, specifically #18) but we should add a test case for it. If that is the issue, then I think it makes sense to switch the default behavior back to `--ignore-platform-reqs`, like we used to have it. 

Otherwise we need to look at this:
https://github.com/dependencies-io/php/blob/090bd104224dc856272659723cc69eeb97156a44/src/collect.php#L68-L73

**Comment by szepeviktor** [comment]
Composer has a *v* test: https://github.com/composer/semver/blob/master/tests/SemverTest.php#L134

**Comment by davegaeddert** [comment]
Right, I'm assuming the issue is the platform reqs but we weren't able to test that on your repo.

**Comment by davegaeddert** [comment]
If you have any interest in trying it, that would be helpful ;)

**Comment by szepeviktor** [comment]
Could I test it on a new branch??

**Comment by davegaeddert** [comment]
We don't have a great way to do that right now. If you put it on your main branch and set the automatic update frequency to "monthly" (in the settings UI on dependencies.io), then no PRs will be created unless you click the update buttons in the UI.

**Comment by davegaeddert** [comment]
Or, you can test it out on a fork of your repo. That's what people usually do if they can't/don't want to commit to the primary repo.
